python/tchannel/thead.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
Suggest moving all the argschemes into a file called `schemes.py` and moving this retry class into a class called `retry.py`.

`thead` is really confusing.